padinfo/padinfo.py

```diff
@@ -1409,3 +1410,20 @@ async def expcurve(self, ctx, start, end: Optional[int], *, query=''):
         state = ClosableEmbedViewState(original_author_id, ClosableEmbedMenu.MENU_TYPE, query,
                                        query_settings, ExperienceCurveView.VIEW_TYPE, props)
         await menu.create(ctx, state)
+
+    @commands.command()
+    async def board(self, ctx, *, message):
+        """Generate a Dawnglare link from the user provided string.
+        Rows/columns separated with spaces.
+        Default fill direction is right then down.
+        Use -1 to invert fill, going down then right."""
+
+        board = BoardGenerator(message)
+
+        if board.invalid_board:
+            await send_cancellation_message(ctx, "An invalid board was defined. Please enter a 5x4, 6x5, or 7x6 "
+                                                 "board. Rows/columns must be separated with spaces.")
```

Comment:
Wait this isn't true, is it? You delete all of the spaces immediately right? (Sorry, this is probably my fault in the ticket - I put spaces as an example to make it clearer to understand but if someone wanted to just put a string without any spaces that should be valid input, and if they wanted to do like `GGG GGG GGG GGG GGG GGG...` that would also be fine. All spaces should get immediately stripped. But I'm pretty sure, you're already doing this.)

Reply:
Having spaces makes it infinitely easier for me to handle the input. I'm stripping the spaces when I convert the input to a list. It doesn't matter for non-inverted, but for inverted it means I don't have to do fancy calculations to split them out into correctly numbered lists for the inverted fill.

It'll probably be easier to handle with some changes I'm making. I'll commit those now for review and if it matters that we need to accept non-spaced inputs I'll work on getting that sorted.

Reply:
Ahhh got it. See if you can figure out a way to split the string based on the board dimensions, but if not, then can you try and require spaces iff it's inverted?

Reply:
The latest commit should resolve this now.